src/parser.py

```
@@ -0,0 +1,73 @@
+import re
+
+def parsear_questoes(texto: str) -> list[dict]:
+    """
+    Analisa o texto extraído de um PDF e o estrutura em uma lista de questões.
+    """
+    questoes = []
+    # Regex para encontrar cada bloco de questão
+    blocos_questoes = re.split(r'(?=\d+\s*–)', texto)[1:]
```

Ruff (0.15.1)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)